playbooks/roles/nginx/vars/main.yml

```diff
@@ -9,3 +9,4 @@ apache_packages_to_remove:
   - apache2-mpm-worker
 
 nginx_systemd_service_path: /etc/systemd/system/nginx.service.d
+nginx_default_html_path: /usr/share/nginx/html
```

Comment:
We shouldn't really be writing to `/usr`. Somewhere in `/var` maybe?

Reply:
Do you think something like `/var/www/streisand-letsencrypt` would be more appropriate?